src/lib/database.ts

```diff
@@ -0,0 +1,130 @@
+import fs from 'fs';
+import { Readable } from 'node:stream';
+import { promisify } from 'node:util';
+import readline from 'readline';
+import yauzl, { Entry } from 'yauzl';
```

Comment:
We already `adm-zip` in the project. Could we please use it here as well to avoid an extra dependency?

Reply:
👍 I'm removing yauzl but not for this reason.

I realized that I don't think we'll have `.zip` files frequently.

FWIW - `adm-zip` can't do stream extract so I can't read just 5 lines of the top of the file. Good for our current purposes, but we might be limited by it in the future.